### PR TITLE
fix: rename old leaflet module to nuxt2-leaflet + update new leaflet module

### PR DESCRIPTION
**Pull request by Gugustinette**
The `nuxt3-leaflet` package was moved to `@nuxtjs/leaflet` : [git](https://github.com/nuxt-modules/leaflet) and [doc](https://leaflet.nuxtjs.org).
I suggest to make it the `leaflet.yml` file and move the previous one (that only supports Nuxt 2) to `nuxt2-leaflet.yml`

I also updated the new package information to point to the new URLs.

### CODE REVIEW

**Comment by Gugustinette** [comment]
Oh well looks like the CI bot don't want me to edit the old `leaflet.yml` 🤕

Can we bypass it ? 🤔

File: modules/nuxt2-leaflet.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: leaflet
```
Comment (danielroe) [inline modules/nuxt2-leaflet.yml:1]:
```suggestion
name: nuxt2-leaflet
```

**Comment by danielroe** [comment]
You need to make sure the `name` and filename match - happy to update again?

**Comment by Gugustinette** [comment]
Made the changes again.

However it seems there's a conflict with the old nuxt3-leaflet that should be deleted.

**Pull request merged**